docs/src/quickstart/index.md

```diff
@@ -57,356 +70,184 @@ for these executables in your `$PATH`.
-➜ cd fuel-indexer/
-
-# Run the test data generator for this example
-➜ cargo run --bin hello-bin
-```
-
-> One contract call will be made, and one event will be emitted to be indexed.
-You can continue running this command to generate more data.
-
-----
-
-## Querying for indexed events
-
-After you've successfully completed all six of the aforementioned steps, you can trigger some test events simply by calling the `new_greeting()` method of your Sway contract. This will produce blocks, transactions, and receipts, which will be emitted by your local Fuel node. These events will be picked up by the indexer and subsequently indexed according to the index that you've deployed. Once you have a few indexed events, you can query the indexer for the data that you wish to receive.
-
-### Query for all records of a type
-
-```sh
-➜ curl -X POST http://127.0.0.1:29987/api/graph/fuel_examples \
+➜ curl -X POST http://0.0.0.0:29987/api/graph/my_project \
    -H 'content-type: application/json' \
-   -d '{"query": "query { greeter { id name first_seen last_seen }}", "params": "b"}' \
+   -d '{"query": "query { tx { id hash status block }}", "params": "b"}' \
 | json_pp
   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                  Dload  Upload   Total   Spent    Left  Speed
 100   364  100   287  100    77   6153   1650 --:--:-- --:--:-- --:--:--  9100
 [
    {
-      "first_seen" : 4,
-      "id" : 2314885811371338051,
-      "last_seen" : 4,
-      "name" : "Ciara"
+      "block" : 7017844286925529648,
+      "hash" : "fb93ce9519866676813584eca79afe2d98466b3e2c8b787503b76b0b4718a565",
+      "id" : 7292230935510476086,
+      "status" : {
+         "block" : "0x8c34daaa2c58629cb98fa66d4f5ce0c0850d24e655ed6006b22204dac42fd918",
+         "status" : "success",
+         "time" : "2022-11-10 14:35:58 UTC"
+      }
    },
    {
-      "first_seen" : 6,
-      "id" : 2314885532299390017,
-      "last_seen" : 6,
-      "name" : "Alex"
+      "block" : 3473793069188998756,
+      "hash" : "5ea2577727aaadc331d5ae1ffcbc11ec4c2ba503410f8edfb22fc0a72a1d01eb",
+      "id" : 4136050720295695667,
+      "status" : {
+         "block" : "0x2b892dd6574e4a803f90c85754d6c8e154ec5f7dd91a25ce962820dce12f15e5",
+         "status" : "success",
+         "time" : "2022-11-10 13:35:58 UTC"
+      }
    },
    {
-      "first_seen" : 8,
-      "id" : 7957705993296504916,
-      "last_seen" : 8,
-      "name" : "Thompson"
+      "block" : 7221293542007912803,
+      "hash" : "d2f638c26a313c681d75db2edfbc8081dbf5ecced87a41ec4199d221251b0578",
+      "id" : 4049687577184449589,
+      "status" : {
+         "block" : "0xa0812af8738da14f7db2f00a53341492aa339f8d88e118820e78a500b11e3560",
+         "status" : "success",
+         "time" : "2022-11-10 12:35:58 UTC"
+      }
    },
-   {
-      "first_seen" : 10,
-      "id" : 2314885530822735425,
-      "last_seen" : 10,
-      "name" : "Ava"
-   }
 ]
 ```
+
+### Finsihed 🥳
```

Comment:
@hrryndrsn 
- I think this sort've leads into something else
- The old Quickstart (the one being replaced in this PR) did a really good job of explaining all of what goes into writing and index (schema, WASM modules, contract, etc) but obviously it was a really long tutorial
- This new Quickstart is much quicker and to the point, but doesn't really explain in-depth the inner workings of what's going on 
- I think this presents the case for a "Deeper dive" section (a separate document/PR) that we can link to here, at the end of this Quickstart. 
  - The user (hopefully) has already seen some success (a deployed index) and now would ideally like to know the specifics about how the components fit together
  - In this "Deep dive" section we could potentially show the user how changes to an indexer could be re-built and redeployed

Reply:
If the point of the quickstart is to show someone an example of how quickly one can get from "zero to query response" with practically no changes to the actual code for the index, then I think that it's best to leave the redeployment step for the book in which it can be placed around additional context; this version of the quickstart doesn't require you to write any code, so if we add a step requiring a user to make a change to the index code, then we should also then explain the code itself before expecting them to make a change that successfully compiles.